### DISCUSSION TITLE
Zeek doesnt capture http on localhost interface on macOS

### DISCUSSION THREAD

**Discussion by BarRozner-Salt** [Security-Onion-Solutions/securityonion#13287] [Unsupported Versions]
Hi,
I have installed and run with the default configs except for the interface, using lo0.
I wanted to test how it works on my local machine but it seems Zeek didnt capture any http traffic (or any traffic or conn for that matter) while my server that runs on my machine is http as plaintext.

Thanks!

**Comment by reyesj2** [comment]
You're running Security Onion on a virtual machine on your mac? 

For ingesting traffic most would use a SPAN port or a TAP to get network traffic into Security Onion
